contracts/lootbox-v1/contracts/Lootbox.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./interfaces/IUniswapV2Router02.sol";
+import "../../dollar/contracts/interfaces/IUbiquityAlgorithmicDollarManager.sol";
+import "../../dollar/contracts/interfaces/IUbiquityAlgorithmicDollar.sol";
+import "./interfaces/IWyvernExchange.sol";
```

Comment:
I think it would be wiser to integrate with an NFT aggregator so that we don't have to reimplement for every NFT marketplace interface 

[https://github.com/ubiquity/ubiquity-dollar-development/discussions/158#discussioncomment-3025880](https://github.com/ubiquity/ubiquity-dollar-development/discussions/158#discussioncomment-3025880)